cmd/subrouter/sr_local_serving_store.go

```
@@ -0,0 +1,297 @@
+	if err != nil {
+		return fmt.Errorf("lock serving-store binding: %w", err)
+	}
+	defer lock.Close()
+	if err := validateLocalServingStoreExpectation(path, expectation); err != nil {
+		return err
+	}
+	client, err := newLocalStoreAttestedClient(&http.Client{}, localBaseURL(), servingStore)
+	if err != nil {
+		return err
+	}
+	response, err := client.Get(strings.TrimRight(localBaseURL(), "/") + "/_subrouter/health")
```

golangci-lint (2.13.2)

[error] (*net/http.Client).Get must not be called. use (*net/http.Client).Do(*http.Request)

(noctx)